changelogs/unreleased/78315_billableSwitchCard.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "BillableSwitchCard: add readonly & required parameters on component",
```

Comment:
Remove on component here too

Reply:
Remove this changelog, 78315.json will do the job for all hr components